encrypted-media-respec.html

```diff
@@ -2547,7 +2557,12 @@
 
         <dt>attribute EventHandler onencrypted</dt>
         <dd>
-          <p>Event handler for the <a def-id="encrypted"></a> event MUST be supported by all HTMLMediaElements as both a content attribute and an IDL attribute.</p>
```

Comment:
Is there a reason you removed the "both" part? This was originally added in https://dvcs.w3.org/hg/html-media/rev/a307fdd7e7bd. @adrianba, do you remember why we needed this?

Related, @cpearce filed https://www.w3.org/Bugs/Public/show_bug.cgi?id=25911 to make sure an IDL attribute reflected the corresponding content attribute. We addressed that specific attribute, but that item was later removed from the spec.

@foolip, do you know if both these items are implied for HTML elements? https://www.w3.org/TR/html5/webappapis.html#event-handlers could be read that way?